### PR TITLE
Improved some codes

### PR DESCRIPTION
**Pull request by jho5245**

### CODE REVIEW

File: src/main/java/org/maxgamer/quickshop/Command/SubCommands/SubCommand_SilentUnlimited.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public void onCommand(@NotNull CommandSender sender, @NotNull String commandLabe
             shop.update();
             MsgUtil.sendControlPanelInfo(sender, shop);
             sender.sendMessage(MsgUtil.getMessage("command.toggle-unlimited", sender,
-                    (shop.isUnlimited() ? plugin.getConfig().getString("command.toggle-unlimited.unlimited") : plugin.getConfig().getString("command.toggle-unlimited.limited"))));
+                    (shop.isUnlimited() ? plugin.getConfig().getString("shop.toggle-unlimited.unlimited") : plugin.getConfig().getString("shop.toggle-unlimited.limited"))));
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/Command/SubCommands/SubCommand_SilentUnlimited.java]: All language phrase should use MsgUtil and store in messages.yml
Config only for config, don't save any phrases.

**Comment by jho5245** [comment]
oops okay

**Comment by jho5245** [comment]
omg code error wut

File: src/main/java/org/maxgamer/quickshop/Util/MsgUtil.java [hunk 2]
```diff
@@ -50,6 +51,7 @@
     private static HashMap<UUID, LinkedList<String>> player_messages = new HashMap<>();
     private static QuickShop plugin = QuickShop.instance;
     private static YamlConfiguration potioni18n;
+    private static DecimalFormat decimalFormat = new DecimalFormat(plugin.getConfig().getString("decimal-format");
```
Comment (Ghost-chu) [inline src/main/java/org/maxgamer/quickshop/Util/MsgUtil.java]: Vault use double, so DecimalFormat is useless in out plugin

Comment (jho5245) [reply src/main/java/org/maxgamer/quickshop/Util/MsgUtil.java]: Some big numbers may look better with comma. isn't it?

**Comment by Ghost-chu** [comment]
You should write the commit message not use defailt Update xxx.
Because git commit message is useful for debug and write changelog.

**Comment by jho5245** [comment]
ok

**Comment by Ghost-chu** [comment]
I have high recommend you use IDEA IDE.
Because you really missed too many ) in code.
Please do not edit code by text editior e.g Notepad++ or VSCode.

**Comment by jho5245** [comment]
I don't know how to load GitHub code the IDE

**Comment by Ghost-chu** [comment]
Run test in your local machine by command `mvn install` before push it.(Require maven install.)

**Comment by Ghost-chu** [comment]
> I don't know how to load GitHub code the IDE

use git tools

**Comment by Ghost-chu** [comment]
There is git guide: https://help.github.com/en/github/using-git

**Comment by jho5245** [comment]
there may not that many people think that adding things like this is useful, but I think that more customizing options, but can be disabled by config, is making the plugin more flexible to use.

**Comment by jho5245** [comment]
this has already merged

**Pull request closed without merging**